src/time_series_parser.jl

```diff
@@ -254,35 +267,36 @@ function TimeSeriesParsedInfo(metadata::TimeSeriesFileMetadata, ta::TimeSeries.T
         metadata.component,
         metadata.name,
         normalization_factor,
-        ta,
+        raw_data,
         metadata.percentiles,
         metadata.data_file,
-        ts_type,
+        metadata.resolution,
+        metadata.time_series_type,
```

Comment:
Can this be removed?

Reply:
`metadata.time_series_type` is used as the constructor for the time_series_data

Reply:
You're passing it to the constructor for TimeSeriesParserInfo. It is unused there.  You should be able to delete this line as well as 217.